MedallionShell.Tests/Streams/MergedLinesEnumerableTestAsync.cs

```diff
@@ -0,0 +1,136 @@
+﻿#if NETCOREAPP2_2_OR_GREATER
```

Comment:
Needed because this is not supported in `net46`.

Reply:
Can remove with 472

Reply:
@madelson Can't move to 472; see https://github.com/madelson/MedallionShell/pull/109/#discussion_r1515366384